m3/data_prepare/report/2-1_collect-sentence-pool.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+import os
+
+import openai
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# Configure OpenAI API client
+openai.api_key = os.getenv("OPENAI_API_KEY")  # Ensure the OpenAI API key is set as an environment variable
+if not openai.api_key:
+    raise ValueError("The OpenAI API key is not set. Please set it as an environment variable 'OPENAI_API_KEY'.")
+
+# Constants
+MODEL_NAME = "meta/llama-3.1-8b-instruct"  # or "meta/llama-3.1-70b-instruct"
```

Comment:
Would be nice to put the link
https://build.nvidia.com/meta/llama-3_1-8b-instruct
To let developers know where does this string come from

Reply:
Thanks. I have added the related information in the scripts.

Reply:
Thanks!